### PR TITLE
CI: Run and report tests on push

### PR DESCRIPTION
**Pull request by nodh**

### CODE REVIEW

**Comment by iaik-jheher** [comment]
CI failing

**Comment by nodh** [comment]
> CI failing

See https://github.com/a-sit-plus/kmm-vc-library/issues/19

**Comment by JesusMcCloud** [comment]
still failing, but in a different way

**Comment by nodh** [comment]
Retrying with updated conventions plugin

File: vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/data/Json.kt [hunk 1]
```diff
@@ -28,3 +30,18 @@ val jsonSerializer by lazy {
...
+    padEncoded = false
+}.build())
+
+
+val Base64Strict = Base64(config = Base64ConfigBuilder().apply {
+    lineBreakInterval = 0
+    encodeToUrlSafe = false
+    isLenient = true
+    padEncoded = true
+}.build())
```
Comment (JesusMcCloud) [inline vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/data/Json.kt:47]: Why is it not used everywhere?

Comment (nodh) [reply vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/data/Json.kt:47]: Isn't this covered in https://github.com/a-sit-plus/kmm-vc-library/pull/18 ?

Comment (JesusMcCloud) [reply vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/data/Json.kt:47]: Sorry, that was very misleading and sloppy! Why don't we want such a shorthand also for Base16?

But come to think of it: Kotlin 1.9 ships with base64 and hex encoding. We might not need the library any more

**Comment by nodh** [comment]
> still failing, but in a different way

Reported at kotest: https://github.com/kotest/kotest/issues/3687

**Comment by nodh** [comment]
I think this PR is ready to merge, please review

**Review by JesusMcCloud — APPROVED** [review]

**Pull request merged**